apps/mobile/src/features/account/components/account-overview-card.tsx

```diff
@@ -4,7 +4,7 @@ import { AccountIcon } from '@/store/accounts/utils';
 import { Box, SkeletonLoader, Text } from '@leather.io/ui/native';
 
 export interface AccountOverviewProps {
-  heading: React.ReactNode;
+  balance: React.ReactNode;
```

Comment:
`heading` still seems like a good name since it can be any react component?

Reply:
I changed it as it's only ever `balance`, I'll just swap it back 👍